### PR TITLE
Run2-alca82 Change output format for MinBias AlCaReco jobs

### PR DESCRIPTION
**Pull request by bsunanda**
Remove the Digi collection and add Raw and Trigger information

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py [hunk 1]
```diff
@@ -9,7 +9,10 @@
         SelectEvents = cms.vstring('pathALCARECOHcalCalMinBias')
     ),
     outputCommands = cms.untracked.vstring(
-	'keep *_gtDigisAlCaMB_*_*',
+        'keep *_hltTriggerSummaryAOD_*_*',
+        'keep *_TriggerResults_*_*',
+        'keep FEDRawDataCollection_rawDataCollector_*_*',
+        'keep FEDRawDataCollection_source_*_*',
```
Comment (arunhep) [inline Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py]: @bsunanda 
how much increase in size of event content and eventually dataset we expect by keeping all this information?

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@arunhep Sorry I took a long time to respond - it takes about 0.59 MB per event for the NZS dataset (phisymmetry studies). This is an increase by a factor of 2 wrt previous output command. However, with the old output command phisysmmetry studies cannot be made using the AlCaReco files.

**Comment by bsunanda** [comment]
@arunhep @franzoni Can you approve this PR

**Comment by bsunanda** [comment]
@arunhep @franzoni Did you converge on this? We have a back up solution to bring back the data volume at the same level as previous time. Should we revert to that. The current AlCaReco o/p's are useless. So we have to introduce this at the earliest.

**Comment by franzoni** [comment]
Dear @bsunanda,

we have a few  questions :

for the phisysmmetry calibration, isn't keeping all HCAL the rechints (as already done in the ALCARECO content you're modifying) sufficient ? We'd imagine that being E-flow based , that's all you need . Please fill us in.

File: Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py [hunk 2]
```diff
@@ -9,7 +9,10 @@
         SelectEvents = cms.vstring('pathALCARECOHcalCalMinBias')
     ),
     outputCommands = cms.untracked.vstring(
-	'keep *_gtDigisAlCaMB_*_*',
```
Comment (franzoni) [inline Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py]: the collection gtDigisAlCaMB of digis which you remove from this PR, is actually L1 trigger digis.
_hltTriggerSummaryAOD_ and _TriggerResults_ might give you the same information you're giving up.

Why do you need to add the full RAW information from all the sub detectors, to make up for the missing L1T digis ?

Comment (bsunanda) [inline Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py]: Olga, one of the main users of the AlCaReco, could not run her codes using the old output commands. So our best bet is to use hltTriggerSummaryAOD and TriggerResults. We can drop the RAW information (which I consulted with Olga) and that is the reason for the increase in data volume. So I am updating this PR.

**Comment by franzoni** [comment]
> We have a back up solution to bring back the data volume at the same level as previous time

Can you elaborate what such "solution" ? 
The size of the relevant 2x ALCARECO for 2017B is (see below) is 1.2 TB, hence not negligible. If 
we need a factor x2 we need to be convinced it's essential... 

[/HcalNZS/Run2017B-HcalCalMinBias-PromptReco-v1/ALCARECO](https://cmsweb.cern.ch/das/request?view=list&limit=50&instance=prod%2Fglobal&input=dataset%3D%2FHcalNZS%2FRun2017B-HcalCalMinBias-PromptReco-v1%2FALCARECO 
)

[/HcalNZS/Run2017B-HcalCalMinBias-PromptReco-v2/ALCARECO](https://cmsweb.cern.ch/das/request?input=dataset%3D%2FHcalNZS%2FRun2017B-HcalCalMinBias-PromptReco-v2%2FALCARECO&instance=prod%2Fglobal)

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py (hunk 2 above)
Comment (franzoni) [inline Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py]: ok, may you please re-do the measurement of event/size on average
after you've updated the event content ?
Thanks!

**Comment by bsunanda** [comment]
The old Data Format used to require 0.21 MB per event and the currently proposed one also needs similar amount (a bit less but not significantly lower). So the overall space requirement will be similar.

**Comment by bsunanda** [comment]
@franzoni Do you have further doubts for this PR?

**Comment by franzoni** [comment]
+1

**Comment by davidlange6** [comment]
merge

**Pull request merged**